README.md

```diff
@@ -95,6 +95,11 @@ Head over to the PAM [style guide][style-guide-link] for documentation and examp
 4.  Push to the branch: `git push origin my-new-feature`
 5.  Submit a pull request
 
+### Building the styleguide
```

Comment:
npm run dev does all of this for you locally. The dist command is only for when it's time To release a new dist package. And then the correct commands are

1. npm run build
2. npm run dist
3. npm version --no-git-tag-version major/minor/patch -m "message"